docs/content/reference/migration/migration-0-13.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Migrating from 0.12
```

Comment:
I suggest making these very explicit, else people jumping version in the future might end up very confused...

This also applies to the existing 0.8->0.9 guide

```suggestion
title: Migrating from 0.12 to 0.13